Cargo.toml

```
@@ -1090,8 +1090,8 @@ polkadot-test-client = { path = "polkadot/node/test/client" }
 polkadot-test-runtime = { path = "polkadot/runtime/test-runtime" }
 polkadot-test-service = { path = "polkadot/node/test/service" }
 polkavm = { version = "0.9.3", default-features = false }
-polkavm-derive = "0.9.1"
-polkavm-linker = "0.9.2"
+polkavm-derive = "0.16.0"
+polkavm-linker = "0.16.0"
```

Even though this also resolved to `0.16.1` we still want to make sure that the buggy version isn't a legal configuration.

```suggestion
polkavm-linker = "0.16.1"
```